### DISCUSSION TITLE
What is this app using for data storage?

### DISCUSSION THREAD

**Discussion by skjuice** [dooit-org/dooit#88] [General]
Cool app, just wondering what it's using to store the data. Is it sqlite or json files or something else? Thanks

**Comment by kraanzu** [comment]
Hey there! Thanks :D

I wanted to use simple txt file with the [todo.txt](https://www.ghacks.net/2019/08/23/what-is-todo-txt-and-how-to-use-it/) format but I think the app is a bit more complex than that
So I maintain a `yaml` file which should be located at `.local/share/dooit/todo.yaml`  in which todos are separated by workspaces and written in todo.txt format :) You can have a look at it

**Comment by skjuice** [comment]
Alright, will do. Thanks for that detail.
